src/main/java/duke/command/DeleteTaskCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.task.Task;
+import duke.Storage;
+import duke.Ui;
+
+import java.io.IOException;
+
+/**
+ * Represents a command to delete a task from the task list.
+ */
+public class DeleteTaskCommand extends Command {
+    private final int taskNum;
+
+    /**
+     * Constructs a DeleteTaskCommand with the specified task number.
+     *
+     * @param taskNum The number of the task to delete.
+     */
+    public DeleteTaskCommand(int taskNum) {
+        this.taskNum = taskNum;
+    }
+
+    /**
+     * Executes the command by deleting the specified task from the task list,
+     * saving the updated task list to storage, and displaying a message indicating
+     * the task has been deleted.
```

Comment:
The method summary seems a bit long to me. Maybe break the parts after the first comma into another sentence?

From our coding standard:
![Screenshot 2024-02-07 at 14 03 49](https://github.com/nus-cs2103-AY2324S2/ip/assets/110611841/25c45da6-c2f6-4111-96f2-71ffe310b506)